src/components/data/LocalVariablesContent.js

```diff
@@ -17,6 +17,27 @@ import { withErrorReporting } from 'src/libs/error'
 import * as StateHistory from 'src/libs/state-history'
 import * as Utils from 'src/libs/utils'
 
+// Exported for testing
+export const getDisplayedAttribute = arr => {
+  const obj = Object.assign({}, ...arr)
+  return [obj.key, obj.value, obj.description ? obj.description : '']
+}
+
+const DESCRIPTION_SUFFIX = '__DESCRIPTION__'
+const isDescriptionKey = k => k.endsWith(DESCRIPTION_SUFFIX)
+
+// Exported for testing
+export const renameAttribute = attr => {
+  const [k, v] = attr
```

Comment:
We usually use object/array deconstructing on input arguments for

- concision
- explicitly pulling out only what's needed
- making it easier to assign default arguments when necessary

```suggestion
export const renameAttribute = ([k, v]) => {
```

Reply:
Done - thanks.